dev/index.jsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react"
+import {render} from "react-dom"
+import {Renderer} from "@khanacademy/perseus/src";
+import {
+    angleQuestion
+} from "@khanacademy/perseus/src/widgets/__testdata__/interactive-graph.testdata";
```

Comment:
🤔  I wonder if we should just use path-relative imports. 🤔  

I guess Vite properly handle these package references, which makes me wonder if we should just be importing like this `import {Renderer} from "@khanacademy/perseus"`.  🤷‍♂️   Or if this works, maybe this is where we start. :)

Reply:
I think relative imports are the way to go. I'd have a bit more confidence that we're actually getting the right code that way 😅 .